src/test/javascript/spec/service/participation.service.spec.ts

```diff
@@ -49,7 +49,7 @@ describe('Participation Service', () => {
         service
             .find(123)
             .pipe(take(1))
-            .subscribe((resp) => expect(resp).toMatchObject({ body: participationDefault }));
+            .subscribe((resp) => expect(resp.body).toMatchObject({ ...participationDefault, results: [], submissions: [] }));
```

Comment:
Might this be another default value issue? Should we consider fixing this in production code?

Reply:
Changed it!

Reply:
It wasn't a default value issue, just the way the initialization of `new StudentParticipation()` works now.